docs/contributing.rst

```diff
@@ -24,8 +24,8 @@ Development Process - Short Version
 #. Work on the master branch for smaller patches and a separate branch for new features.
 #. Make changes, ``git add`` and then commit. Make sure to link the issue number in the commit message.
 #. Run the following commands: ``pylint browser_history``, ``pytest --cov=./browser_history``
-#. (Optional) If you're updating the documentation, run the following: ``cd docs``, ``make html``
-   and then open ``_build/html/index.html`` in a browser to confirm that the documentation rendered correctly.
+#. (Optional) If you're updating the documentation, Update ``docs/quickstart.rst`` and ``README.md`` simoultaneously.
```

Comment:
Spelling xD

Reply:
I've become the very thing I sworn to destroy xD